### PR TITLE
Upgrade to Flow 0.136.0

### PR DESCRIPTION
**Pull request by mwiencek**
It's now an error for generics to escape outside of the scope they're defined in. Which usually just means that if a variable's type contains a generic type variable, then we can't pass that variable around without also passing the associated generic.

### CODE REVIEW

File: root/layout/components/Search.js [hunk 1]
```diff
@@ -46,34 +54,39 @@ const TYPE_OPTION_GROUPS = [
...
-            value={key}
-          >
-            {text}
-          </option>
-        );
-      })
-    ))}
+    {TYPE_OPTION_GROUPS.map((group) => (
+      Object.entries(group)
+        // $FlowIssue[incompatible-call]
```
Comment (reosarevok) [inline root/layout/components/Search.js:75]: Can't group be typed to avoid this?

**Review by reosarevok — APPROVED** [review]

File: root/layout/components/Search.js (hunk 1 above)
Comment (mwiencek) [reply root/layout/components/Search.js:75]: It's already typed, but the issue is that `Object.entries` is hard-coded to return `Array<[string, mixed]>`. You can't use `mixed` without either refining it or casting it to `any` first, and both of those are worse solutions.

Comment (reosarevok) [reply root/layout/components/Search.js:75]: Well, that sucks :(

Comment (mwiencek) [reply root/layout/components/Search.js:75]: I suspect they'll fix the type eventually (I see they're working on improving generics checking as we speak). So the error suppression isn't so bad in the meantime: unlike `any` it doesn't prevent marking the file as strict, it only affects `incompatible-call` and not other kinds of errors, and we can run Flow with `--include-warnings` to find unneeded error suppression comments in the future.

**Review by yvanzo — APPROVED** [review]

**Pull request merged**